### DISCUSSION TITLE
remove react-native-ratings

### DISCUSSION THREAD

**Discussion by hsavit1** [react-native-elements/react-native-elements#3431] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Explain what you did

Running the project with react-native-web

### Expected behavior

Looked into the lighthouse score, realized that react-native-ratings contains lodash

### Describe the bug

React native ratings contains a lodash dependency, which has quite a large impact on web. 

### Steps To Reproduce

```markdown
run web project with latest 3.4 release of RNE, see lighthouse score ways to improve time to first load
```

**Comment by hsavit1** [comment]
this project can live on its own. I don't think it needs to be included in react-native-elements

**Reply by arpitBhalla** [reply]
@kedar09 what do you think, should we remove this component?

**Reply by kedar09** [reply]
@arpitBhalla I will check that and let you know

**Reply by kedar09** [reply]
@arpitBhalla I think this PR https://github.com/Monte9/react-native-ratings/pull/122 will solve this problem, so no need to remove react-native-ratings.

**Reply by arpitBhalla** [reply]
that PR is way too old, also the package seems to be less maintained, & it's a separate package, anyone can install it if they want, Lets remove this.

**Comment by hsavit1** [comment]
another option is to fork the library and get rid of lodash. It literally only uses it in 1 file for 2 functions, maps and times

https://github.com/Monte9/react-native-ratings/blob/master/src/TapRating.tsx#L144
